src/plonemeeting/portal/core/vocabularies.py

```
@@ -186,13 +206,18 @@ def __call__(self, context):
         url = get_api_url_for_meetings(institution)
         if not url:
             return SimpleVocabulary([])
-        response = requests.get(url, auth=(institution.username, institution.password), headers=API_HEADERS)
+        response = requests.get(
```

Ruff (0.15.14)

[error] Probable use of `requests` call without timeout

(S113)